pipert/utils/logger_utils.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+from logging.handlers import SocketHandler, TimedRotatingFileHandler
+
+
+def create_parent_logger(name):
+    logger_type = os.environ.get('LOGGER', 'file').lower()
+    return getattr(LoggerGenerator, f"_create_{logger_type}_logger")(name)
+
+
+class LoggerGenerator:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def _create_cutelog_logger(name):
+        logger = logging.getLogger(name)
+        logger.setLevel(logging.DEBUG)
+        logger.propagate = False
+        socket_handler = SocketHandler('127.0.0.1', 19996)
+        socket_handler.setFormatter(logging.Formatter(
+            "%(asctime)s - %(levelname)s - %(name)s - %(message)s"))
+        logger.addHandler(socket_handler)
+        return logger
+
+    @staticmethod
+    def _create_file_logger(name):
+        class FileLogger(logging.Logger):
+            def __init__(self, name, level=logging.NOTSET):
+                super().__init__(name=name, level=level)
+
+            def getChild(self, suffix: str) -> logging.Logger:
```

Comment:
If you're going to use statically defined type declarations at least be consistent and use it everywhere.
Also `getChild` should be `get_child`.

Reply:
i copied it from the logging.Logger class, it is important that the function will have the same name and signature so i used its declarations as well.

Reply:
Still you should be consistent throughout the file